tokio/src/net/tcp/listener.rs

```diff
@@ -12,8 +12,9 @@ use std::task::{Context, Poll};
 cfg_tcp! {
     /// A TCP socket server, listening for connections.
     ///
-    /// Also implements a stream over the connections being received on this listener.
-    ///
+    /// You can accept a new connection by using the accept method. Alternatively this type
+    /// implements the `Stream` trait, which allows you to use the listener in places that want a
+    /// stream.
```

Comment:
This should probably wrap with the text on the following line.

Reply:
We might as well also make `accept` and `Stream` links.